test/unit/test.speech_to_text.v1.js

```diff
@@ -344,4 +344,147 @@ describe('speech_to_text', function() {
       assert(isStream(speech_to_text.createRecognizeStream()));
     });
   });
+
+  describe('asynchronous callback api', function() {
+    it('should register new callback url', function() {
+      const path = '/v1/register_callback?callback_url=http%3A%2F%2Fwww.callback.fr%2Fresults&user_secret=ThisIsMySecret';
+      const response = {
+        status: 200,
+        url: 'http://www.callback.fr/results'
+      };
+
+      nock(service.url).persist().post(path).delay(100).reply(200, response);
```

Comment:
You don't need to call `.persist()` here, that's for setting up a single response to handle multiple tests.